dictionaries/mn_MN/src/hunspell/download.js

```diff
@@ -0,0 +1,35 @@
+import fetch from 'node-fetch';
+import { writeFile } from 'fs/promises';
+
+const files = [
+    {
+        url: 'https://raw.githubusercontent.com/bataak/dict-mn/main/mn_MN/mn_MN.aff',
+        output: 'index.aff'
```

Comment:
Since we are downloading to the same directory, we can use `import.meta.url` as the base to get an absolute file URL.

```suggestion
        output: new URL('index.aff', import.meta.url)
```